app/components/Form/ApplicationFormStatus.tsx

```diff
@@ -28,7 +28,7 @@ const StatusContainer = styled('div')`
   align-items: flex-end;
 `;
 
-const SavingStatusContainer = styled('div')`
+const SavingStatusContainer = styled('header')`
```

Comment:
Sorry, I meant that the flexbox in `ApplicationForm` should be the one turned into a header, not this one, since it contains the `h1` element.